frontend/awx/resources/templates/WorkflowVisualizer/WorkflowVisualizer.tsx

```diff
@@ -30,7 +32,17 @@ export function WorkflowVisualizer() {
   const toolbarActions = useWorkflowVisualizerToolbarActions(wfNodes?.results ?? []); // The argument for this function will need to be the number of nodes.
   let topologyScreen;
   if (!wfNodes) {
-    topologyScreen = <div>Loading...</div>;
+    topologyScreen = (
+      <EmptyState>
+        <EmptyStateHeader
+          titleText={t('Please wait until the Workflow Visualizer is populated.')}
+          headingLevel="h4"
+          icon={<ShareAltIcon />}
```

Comment:
The icon in the [mockup](https://www.sketch.com/s/46cfe983-742c-4843-a462-6bd7bc04d8e1/a/8VAdzqY#Comment) is grey and large 

```suggestion
          icon={
            <Icon size="xl" style={{ color: getPatternflyColor('grey') }}>
              <ShareAltIcon />
            </Icon>
          }
```
<img width="536" alt="Screenshot 2023-11-03 at 1 57 54 PM" src="https://github.com/ansible/ansible-ui/assets/15881645/c885f52b-b981-48a4-a9ca-b312ea623266">